src/generate.cc

```diff
@@ -4815,8 +4702,7 @@ static void fill_treasure(int x1, int x2, int y1, int y2, int difficulty)
 				}
 				else if (value < 50)
 				{
-					/* Trap */
-					place_trap(y, x);
+					/* Do nothing */
```

Comment:
I wonder what effect removing this if branch would have...

Soooo many ad-hoc magic numbers :(